aws/resource_aws_appmesh_virtual_router.go

```diff
@@ -76,7 +80,7 @@ func resourceAwsAppmeshVirtualRouterCreate(d *schema.ResourceData, meta interfac
 	req := &appmesh.CreateVirtualRouterInput{
 		MeshName:          aws.String(d.Get("mesh_name").(string)),
 		VirtualRouterName: aws.String(d.Get("name").(string)),
-		Spec:              expandAppmeshVirtualRouterSpec(d.Get("spec").([]interface{})),
+		Spec:              &appmesh.VirtualRouterSpec{},
```

Comment:
`Spec` still needs to be specified here even though it can only be empty.

Reply:
Just a note that this is where the new VirtualRouter `listeners` change goes for https://github.com/awslabs/aws-app-mesh-examples/issues/94, which is a required field in the new APIs.